src/pages/Swap/AdvancedSwapMode/PairDetails/PairDetails.component.tsx

```diff
@@ -16,6 +16,17 @@ interface PairDetailsProps {
   activeCurrencyOption: Token | null | undefined
 }
 
+const RelativePrice = ({ relativePrice }: { relativePrice: string }) => {
```

Comment:
```suggestion
const RelativePrice = ({ price }: { price: string }) => {
```

The relative price function receives a price,  and returns a relative price. So I think price, is a better name for the argument.